crankshaft-console/src/view/tasks.rs

```diff
@@ -0,0 +1,65 @@
+//! Renders the tasks view.
+use ratatui::Frame;
+use ratatui::layout::Constraint;
+use ratatui::layout::Direction;
+use ratatui::layout::Layout;
+use ratatui::style::Color;
+use ratatui::style::Modifier;
+use ratatui::style::Style;
+use ratatui::widgets::Block;
+use ratatui::widgets::Borders;
+use ratatui::widgets::Cell;
+use ratatui::widgets::Row;
+use ratatui::widgets::Table;
+
+use crate::state::task::TuiTasksState;
+
+/// Renders the tasks view.
+pub(crate) fn render_tasks(frame: &mut Frame<'_>, tasks_state: &TuiTasksState) {
+    let area = Layout::default()
+        .direction(Direction::Vertical)
+        .margin(1)
+        .constraints([Constraint::Min(0)])
+        .split(frame.size());
+
+    let headers = ["Task ID", "Event Type", "Timestamp", "Message"]
+        .iter()
+        .map(|h| {
+            Cell::from(*h).style(
+                Style::default()
+                    .fg(Color::Yellow)
+                    .add_modifier(Modifier::BOLD),
+            )
+        });
+    let header = Row::new(headers).height(1).bottom_margin(1);
+
+    let rows = tasks_state.tasks().values().map(|task| {
+        let (timestamp, message) = task
+            .latest_log()
+            .map(|log| (log.timestamp.to_string(), log.message.clone()))
+            .unwrap_or_else(|| ("-".into(), "-".into()));
+
+        let row = vec![
```

Comment:
Same comment as above: we might be able to get away with using an array here rather than `Vec`.